### PR TITLE
Addresses merge bug #1126

### PR DESCRIPTION
**Pull request by cjmyers**

### CODE REVIEW

**Comment by cjmyers** [comment]
Fixes #1126

File: lib/views/submit.js [hunk 1]
```diff
@@ -349,7 +349,7 @@ function submitPost (req, res) {
           }
           res.send(pug.renderFile('templates/views/errors/errors.jade', locals))
         }
-      })
+      }) */
```
Comment (udp) [inline lib/views/submit.js]: How come this is commented out?

Comment (cjmyers) [reply lib/views/submit.js]: It seems to be causing an unhandled promise rejection.  Actually, I'm still seeing unhandled promise rejections when you try to merge and are prevented due to trying to submit non-identical objects into an existing collection, but this seemed to move it closer to where it was happening.  It is strange. Still bad at promises

Comment (udp) [reply lib/views/submit.js]: Refactoring to use async/await should make it much more clear where exceptions are going.  Will accept to solve the immediate issue but we need to sort this soon.

**Comment by cjmyers** [comment]
Agreed.

**Pull request merged**